.github/workflows/PullRequestBuild.yml

```
@@ -62,36 +66,28 @@ jobs:
       # 上传构建产物
       - name: Upload build artifacts { ${{ env.Version }} With Console AllInOne}
         uses: actions/upload-artifact@v4
-        env:
-          GITHUB_TOKEN: ${{ secrets.RELEASE_TOKEN }}
         with:
           name: ${{ env.RCVersion }}  ${{ env.CommitMessage }}  [ Auto Release ]
-          path: ./MusyncSaveDecode/Archive/MusyncSaveDecode_WithConsole_${version}_AllInOne.zip
+          path: ./MusyncSaveDecode/Archive/MusyncSaveDecode_WithConsole_${{version}}_AllInOne.zip
```

actionlint (1.7.12)

[error] undefined variable "version". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)